tests/common/mod.rs

```diff
@@ -11,17 +12,17 @@ use std::{
 };
 use uniffi::TargetLanguage;
 
-lazy_static! {
-    pub static ref CLIENT: Arc<SaasShield> = {
-        let config = SaasShieldConfiguration::new(
-            "http://localhost:32804".to_string(),
-            "0WUaXesNgbTAuLwn".to_string(),
-            false,
-            Some(1.1),
-        )
-        .unwrap();
-        SaasShield::new(&config)
-    };
+pub type TestResult = Result<(), AlloyError>;
+
+pub fn get_client() -> Arc<SaasShield> {
+    let config = SaasShieldConfiguration::new(
+        "http://localhost:32804".to_string(),
+        "0WUaXesNgbTAuLwn".to_string(),
+        false,
+        Some(1.1),
+    )
+    .unwrap();
+    SaasShield::new(&config)
```

Comment:
Why switch to making a new client for every test? Were you running into issues with the shared client?

Reply:
I got the `dropped dispatch` error more and more as more tests were added. So this fixes that entirely.